tests/test_exceptions.py

```diff
@@ -42,6 +42,12 @@ def test_get_http_status_code(self):
             == 400
         )
 
+    def test_invalid_parameter_value(self):
+        mlflow_exception = MlflowException.invalid_parameter_value("test")
+        deserialized_exception = json.loads(mlflow_exception.serialize_as_json())
+        assert deserialized_exception["message"] == "test"
+        assert deserialized_exception["error_code"] == INVALID_PARAMETER_VALUE
```

Comment:
```suggestion
        mlflow_exception = MlflowException.invalid_parameter_value("test")
        assert mlflow_exception.error_code == "INVALID_PARAMETER_VALUE"
```

Can we directly assert the `error_code` attribute?